WinUIGallery/ControlPages/DesignGuidance/AccessibilityKeyboardPage.xaml

```diff
@@ -1,387 +1,681 @@
+                    <Button
+                        Grid.Row="2"
+                        Grid.Column="2"
+                        HorizontalAlignment="Stretch"
+                        Content="Not a stop"
+                        IsTabStop="False" />
+                </Grid>
+            </core:ControlExample.Example>
+            <core:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;Grid RowSpacing="8" ColumnSpacing="8"&gt;
+    &lt;Grid.RowDefinitions&gt;
+	    &lt;RowDefinition Height="Auto"/&gt;
+	    &lt;RowDefinition Height="Auto"/&gt;
+	    &lt;RowDefinition Height="Auto"/&gt;
+    &lt;/Grid.RowDefinitions&gt;
+    &lt;Grid.ColumnDefinitions&gt;
+	    &lt;ColumnDefinition Width="Auto"/&gt;
+	    &lt;ColumnDefinition Width="Auto"/&gt;
+	    &lt;ColumnDefinition Width="Auto"/&gt;
+    &lt;/Grid.ColumnDefinitions&gt;
+
+    &lt;TextBlock Grid.Column="1" HorizontalAlignment="Center" Text="Column 1"/&gt;
+    &lt;TextBlock Grid.Column="2" HorizontalAlignment="Center" Text="Column 2"/&gt;
+
+    &lt;!--
+        This example is a bit contrived, since you could fix the
+        tab order by reordering the elements in the XAML, but
+        sometimes that is not easy to do.
+    --&gt;
+    &lt;TextBlock Grid.Row="1" VerticalAlignment="Center" Text="Row 1"/&gt;
+    &lt;Button Grid.Column="1" Grid.Row="1" Content="First stop" TabIndex="1" HorizontalAlignment="Stretch"/&gt;
+    &lt;Button Grid.Column="2" Grid.Row="1" Content="Third stop" TabIndex="3" HorizontalAlignment="Stretch"/&gt;
+
+    &lt;TextBlock Grid.Row="2" VerticalAlignment="Center" Text="Row 2"/&gt;
+    &lt;Button Grid.Column="1" Grid.Row="2" Content="Second stop" TabIndex="2" HorizontalAlignment="Stretch"/&gt;
+    &lt;Button Grid.Column="2" Grid.Row="2" Content="Not a stop" IsTabStop="False" HorizontalAlignment="Stretch"/&gt;
+&lt;/Grid&gt;
+                    </x:String>
+            </core:ControlExample.Xaml>
+        </core:ControlExample>
+
+        <TextBlock
+            Margin="0,20,0,0"
+            Style="{ThemeResource SubtitleTextBlockStyle}"
+            AutomationProperties.HeadingLevel="Level2"
+            Text="Arrow keys" />
+
+        <RichTextBlock>
+            <Paragraph>
+                Users expect groups of similar, related controls to be navigable via <Bold>Arrow keys</Bold>, too. This can be instead of
+                <Italic>or</Italic> in addition to tab navigation, depending on the situation.
+                <LineBreak />
+            </Paragraph>
+            <Paragraph>
+                Groups of controls that support arrow key navigation typically support Home/End and PgUp/PgDn, too.
+                <LineBreak />
+            </Paragraph>
+            <Paragraph>
+                See
+                <Hyperlink
+                    NavigateUri="https://learn.microsoft.com/windows/apps/design/input/keyboard-interactions#navigation">
+                    Keyboard interactions#Navigation

```

Comment:
Did you mean to include the # inside the displayed text of the link? Maybe we use - instead if we want to indicate that we are only linking to sections of the same page?

Reply:
I *did* deliberately chooose `#` as the separator. It's a bit dev-y, but my reasoning was that `#` represents an anchor link---it's the literal character used to denote anchors in a URL. I worried `-` would be too ambiguous (is this a subpage of a section? A long title? etc).

I'm open to changing it, though, if it's confusing.

Reply:
The only thing I'm afraid is people thinking this is by mistake though I think this is a risk we can take here. After all, our audience is mostly devs.